R/archive.R

```diff
@@ -181,7 +181,10 @@ epi_archive =
             cat(sprintf("Data archive (stored in DT field): %i x %i\n",
                         nrow(self$DT), ncol(self$DT)))
             cat("----------\n")
-            cat(sprintf("Public methods: %s",
+            cat(sprintf("Column names in DT: %s\n", 
+                        paste(colnames(self$DT), collapse = ", ")))
```

Comment:
One potential issue here: what if the DT has like 80 columns? Maybe print the first 10 (or something) along with "and 70 more columns". Also, this could easily run on, so there needs to be some returns.